sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/AppCenterIngestion.java

```diff
@@ -175,7 +175,7 @@ public void onBeforeCalling(URL url, Map<String, String> headers) {
                 }
                 String authToken = logHeaders.get(AUTHORIZATION_HEADER);
                 if (authToken != null) {
-                    logHeaders.put(AUTHORIZATION_HEADER, HttpUtils.hideSecret(authToken));
+                    logHeaders.put(AUTHORIZATION_HEADER, String.format(AUTH_TOKEN_FORMAT, "***"));
```

Comment:
Nitpick: I think it would be better to add a helper function to `HttpUtils` since there we already have `hideSecret`, `hideTickets`, etc.